eyeshade/controllers/reports.js

```diff
@@ -484,7 +527,8 @@ module.exports.routes = [
   braveHapi.routes.async().path('/v2/reports/publishers/statements').config(v2.publishers.statements),
   braveHapi.routes.async().path('/v1/reports/publishers/status').config(v1.publishers.status),
   braveHapi.routes.async().path('/v2/reports/publishers/status').config(v2.publishers.status),
-  braveHapi.routes.async().path('/v1/reports/surveyors/contributions').config(v1.surveyors.contributions)
+  braveHapi.routes.async().path('/v1/reports/surveyors/contributions').config(v1.surveyors.contributions),
```

Comment:
if `standard` lint would allow leading commas, our reviews would be half as large...

Reply:
you need a `runtime.queue.create('report-grants-outstanding`) in `module.exports.initialize()`